dracut/30ignition/ignition-setup-base.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -euo pipefail
+
+copy_file_if_exists() {
+    src="${1}"; dst="${2}"
+    if [ -f "${src}" ]; then
+        echo "Copying ${src} to ${dst}"
+        cp "${src}" "${dst}"
+    else
+        echo "File ${src} does not exist.. Skipping copy"
+    fi
+}
+
+destination=/usr/lib/ignition/
+mkdir -p $destination
+
+# We will support grabbing a platform specific base.ign config
+# from the initrd at /usr/lib/ignition/platform/${PLATFORM_ID}/base.ign
+copy_file_if_exists "/usr/lib/ignition/platform/${PLATFORM_ID}/base.ign" "${destination}/base.ign"
```

Comment:
What is `$PLATFORM_ID` defined by?

Reply:
`ignition-generator` writes it into `/run/ignition.env`, which is added to the environment by `ignition-setup-base.service`.